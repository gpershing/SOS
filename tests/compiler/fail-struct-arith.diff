diff: ./tests/compiler/tests/compiler/fail-struct-arith.err: No such file or directory
