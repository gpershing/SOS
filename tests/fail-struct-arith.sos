struct s1 = {int a, float b}

struct s2 = {int a, int b}

s1 + s2
