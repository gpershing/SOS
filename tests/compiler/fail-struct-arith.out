Can only add or subtract structs of matching type
